jekyll/_cci2_ja/config-overview.md

```diff
@@ -5,7 +5,7 @@ description: "CircleCI の設定ファイルの概要を説明します。"
 categories:
   - 設定
 order: 2
-redirect_from: /2.0/config-start/
+redirect_from: /ja/2.0/config-start/
```

Comment:
I can see that accessing to the following will redirect to Japanese page. https://circleci.com/docs/2.0/config-start/
![Jun-09-2022 13-17-08](https://user-images.githubusercontent.com/6015450/172763219-ba6efc14-d4cc-4fbf-a8c4-9709cb2e3f2a.gif)